src/transformState.js

```diff
@@ -5,7 +5,35 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    for (const key in obj) {
```

Comment:
you don't need a second for here. an action is an object and you know all its properties. you just need to check `action.type`